queries.yaml

```diff
@@ -1779,9 +1832,9 @@ post_process:
   # process may also have created small inners.
   - fn: vectordatasource.transform.drop_small_inners
     params:
-      end_zoom: 15
+      end_zoom: 17
       source_layers: [buildings]
-      pixel_area: 0.25
+      pixel_area: 10.0
```

Comment:
Dropping small inners looks like this, where the pink areas are removed. Looks good to me!

![image](https://user-images.githubusercontent.com/853051/152888555-59fec8be-2ec0-433d-81ff-fc302ce5f5ed.png)

Reply:
Actually this is changing zoom 17 in error (so zoom 16 in 512)?

This courtyard should still be present, but is not.

![image](https://user-images.githubusercontent.com/853051/153309487-e1118cc0-1635-4d40-a052-dc87c1e21b7c.png)

Reply:
Same thing is happening in Barcelona at zoom 17 (with zoom 16 tile in 512px space):

Most these courtyards are no longer present:

<img width="416" alt="image" src="https://user-images.githubusercontent.com/853051/153312765-3b9f3bce-0aed-485e-bd49-5fffebc90e9f.png">

Reply:
made a change on tilequeue let's see whether that can fix it: tilezen/tilequeue#407